query-node/mappings/src/storage/index.ts

```diff
@@ -585,3 +599,89 @@ async function removeDistributionBucketOperator(store: DatabaseManager, operator
     }
   }
 }
+
+export async function processSetNodeOperationalStatusMessage(
+  store: DatabaseManager,
+  event: SubstrateEvent,
+  workingGroup: WorkingGroup,
+  meta: ISetNodeOperationalStatus
+): Promise<void> {
+  if (workingGroup.name !== 'distributionWorkingGroup' && workingGroup.name !== 'storageWorkingGroup') {
+    return invalidMetadata(`The ${workingGroup.name} is incompatible with the remarked setNodeOperationalStatus`)
+  }
+
+  const workerId = Number(meta.workerId)
+  const bucketId = String(meta.bucketId)
+
+  if ((await getWorker(store, workingGroup.name, workerId)) === undefined) {
+    return invalidMetadata(`The worker ${workerId} does not exist in the ${workingGroup.name} working group`)
+  }
```

Comment:
I'm wondering, do we need to bother with worker for storage buckets? Those only have a single operator so I guess the worker ID doesn't make any difference?